src/java/org/apache/cassandra/db/filter/ColumnFilter.java

```diff
@@ -483,84 +549,56 @@ public boolean equals(Object other)
     @Override
     public String toString()
     {
-        if (fetchAllRegulars && queried == null)
-            return "*";
+        String prefix = "";
 
-        if (queried.isEmpty())
-            return "";
+        if (fetchAllRegulars && queried == null)
+            return "*/*";
 
-        Iterator<ColumnMetadata> defs = queried.selectOrderIterator();
-        if (!defs.hasNext())
-            return "<none>";
+        if (fetchAllRegulars && fetchAllStatics)
+            prefix = "*/";
 
-        StringBuilder sb = new StringBuilder();
-        while (defs.hasNext())
+        if (fetchAllRegulars && !fetchAllStatics)
         {
-            appendColumnDef(sb, defs.next());
-            if (defs.hasNext())
-                sb.append(", ");
+            prefix = queried.statics.isEmpty()
+                     ? "<all regulars>/"
+                     : String.format("<all regulars>+%s/", columnsToString(queried.statics::selectOrderIterator));
```

Comment:
I wonder if it would be easier to understand if we simply print the specific fetched columns, so the only abbreviation used by `toString` is `*`.